compute_tools/src/compute.rs

```diff
@@ -1134,8 +1137,17 @@ impl ComputeNode {
         let files = fs::read_dir(core_dump_dir)?;
         let cores = files.filter_map(|entry| {
             let entry = entry.ok()?;
-            let _ = entry.file_name().to_str()?.strip_prefix("core.")?;
-            Some(entry.path())
+
+            let is_core_dump = match entry.file_name().to_str()? {
+                n if n.starts_with("core.") => true,
+                n if n == "core" => true,
+                _ => false,
+            };
+            if is_core_dump {
```

Comment:
This can also be expressed as `is_core_dump.then(|| entry.path())`

Reply:
matter of taste but I find this more readable as an if-else, even though it's more verbose